packages/components/src/Dialog/Layout/DialogLayout.story.tsx

```diff
@@ -26,45 +26,39 @@
 
 import React from 'react'
 import { Story } from '@storybook/react/types-6-0'
-import { Constitution, ConstitutionShort } from '../../__mocks__/Constitution'
+import { ConstitutionShort } from '../../__mocks__/Constitution'
 import { Box } from '../../Layout'
-import { DialogContent, DialogLayout } from '.'
+import { Button, ButtonTransparent } from '../../Button'
+import { DialogContent, DialogFooter, DialogLayout } from '.'
 
 export interface DialogLayoutExampleProps {
   header?: string
   footer?: string
-  length?: 'short' | 'long'
 }
 
-const Template: Story<DialogLayoutExampleProps> = ({
-  header,
-  footer,
-  length,
-}) => (
+const Template: Story<DialogLayoutExampleProps> = ({ header, footer }) => (
   <Box bg="ui1">
     <DialogLayout
       footer={footer && 'Footer text'}
       header={header && 'Header text'}
     >
-      {length === 'long' ? <Constitution /> : <ConstitutionShort />}
+      <ConstitutionShort />
     </DialogLayout>
   </Box>
 )
 
-export const Basic = Template.bind({})
-Basic.args = {
-  length: 'short',
-}
+export const Content = Template.bind({})
+Content.args = {}
```

Comment:
Any specific reason to rename this story? Most of our `.story` files start with a story named "Basic" so it'd be nice to keep with that convention unless there's a specific reason not to do so.

Also the `length` option made it easier to compose the stories below - wondering why you choose to remove that functionality and hard-code it instead?

```suggestion
export const Basic = Template.bind({})
Basic.args = {
  length: 'short'
}
```

Reply:
the length option is not used anywhere as `long` so I didn't see the point of having it.